inblay/i18n/es.po

```diff
@@ -268,11 +269,26 @@ msgstr "Talla de camisa"
 msgid "Significant Other"
 msgstr "Pareja"
 
+#. module: inblay
+#: model_terms:ir.ui.view,arch_db:inblay.report_purchaseorder_document_inherit
+msgid "Subtotal"
+msgstr "Subtotal"
```

Comment:
If the translation is equal to the original string, it's best to leave empty, because it will  be ignored and removed on next translation export

Reply:
Done.